perf-test.yaml

```diff
@@ -0,0 +1,71 @@
+trigger: none
+pr: none
+
+stages:
+- stage: Smallfiles
+  jobs:
+    - job: PerformanceTest
+      timeoutInMinutes: 360
+      strategy:
+        matrix:
+          Ubuntu-22:
+            imageName: "azcopyPerfUbuntu22"
+            Description: "AzCopy Perf Test"
+
+      pool:
+        name: "AzCopyPerfTestUbuntu"
+        demands:
+          - ImageOverride -equals $(imageName)
+
+      variables:
+      - group: AzCopyPerfTestTargets
+
+      steps:
+        - script: |
+            echo $(Description)
+            hostnamectl
+          displayName: 'Print Agent Info'
+
+        - task: GoTool@0
+          inputs:
+            version: '1.19.3'
+
+        - script: |
+            go build -o $GOROOT/bin/azcopy
+            azcopy --version
+          displayName: 'Build Azcopy'
+          env:
+            AZCOPY_AUTO_LOGIN_TYPE: $(AZCOPY_AUTO_LOGIN_TYPE)
+            AZCOPY_MSI_CLIENT_ID: $(AZCOPY_MSI_CLIENT_ID)
+            AZCOPY_CONCURRENCY_VALUE: "256"
+            AZCOPY_SHOW_PERF_STATES: "1"
+
+        - script: |
+            time azcopy copy $(Blob2BlobLargeFilesSrc) $(Blob2BlobLargeFilesDst) --recursive --block-size-mb=128 --log-level=ERROR --cap-mbps=40000
```

Comment:
Can we parse the output of time and put it in some file for all three test cases and then upload this file in our artifacts.

Reply:
I'm looking for a better way to do it

Reply:
you can refer to blobfuse2 for doing this as well